dashboard/templates/dashboard/articles/review.html

```diff
@@ -8,6 +8,8 @@
 {% block content %}
 <div class="review-page-container">
     <div class="answer-area">
+        {% include 'snippets/show_messages.html' %}
+        {% endif %}
```

Comment:
This seems to be a stray `endif`. Can you check?

Reply:
Fixed